base/src/main/java/com/thoughtworks/go/util/SystemEnvironment.java

```diff
@@ -605,7 +605,7 @@ public String getBaseUrlForShine() {
     }
 
     public Boolean isShineEnabled() {
-        String shineEnabled = getEnvironmentVariable("SHINE_ENABLED");
+        String shineEnabled = getEnvironmentVariable("SHINE_ENABLED", "false");
         return shineEnabled == null ? false : !"false".equalsIgnoreCase(shineEnabled); //should return true for shine_enabled set to anything but false.
```

Comment:
Change this line to return ` !"false".equalsIgnoreCase(shineEnabled);` ?

Reply:
This code is going away soon :) I don't want to give it too much love ;)